src/app/framework-modules/elements/ct-field-generator/component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input } from '@angular/core';
+import { FormControl} from '@angular/forms';
+import { Form } from '../../formdata/form/form';
+import { FormData } from '../../formdata/form/formData';
+
+@Component({
+	selector: 'ct-field-generator',
+	templateUrl: './component.html',
+	styleUrls: []
+})
+
+/**
+ * Automated field Generator component
+ * @fd - Formdata
+ */
+export class CTFieldGenerator {
+	@Input() public fd: FormData;
+
+	fields:any;
+	form:Form;
+	control:FormControl;

```

Comment:
Still wondering if you have a linter enabled.  This formatting is inconsistent with the above:
```
fd: FormData;
```
In one place there is a space after the colon, in the other place there is no space after the colon.